repository/pom.xml

```diff
@@ -504,6 +493,12 @@
           <artifactId>jira-workitem</artifactId>
           <type>zip</type>
           <scope>provided</scope>
+          <exclusions>
+            <exclusion>
+              <groupId>org.jvnet</groupId>
+              <artifactId>mimepull</artifactId>
+            </exclusion>
+          </exclusions>
```

Comment:
Not sure about this one.

Reply:
if the scope is provided it should not be a big deal as long as jira-workitem should have that dependency as compiled already.

Reply:
Removed in 3c1e362